### PR TITLE
lps-96749

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:stop

**Comment by lawrence-lee** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-service/service.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.2.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_2_0.dtd">
 #end
 
-<service-builder #if (${liferayVersion.startsWith("7.2")})dependency-injector="ds"#end package-path="${package}">
+<service-builder #if (${liferayVersion.startsWith("7.2")} && (${dependencyInjector} == "ds"))dependency-injector="ds"#end package-path="${package}">
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-service/service.xml:10]: hey @lawrence-lee   we need to support `dependency-injector="spring"` path.

File: modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/ProjectTemplatesArgs.java [hunk 2]
```diff
@@ -223,6 +231,12 @@ protected boolean isList() {
 	)
 	private String _contributorType;
 
+	@Parameter(
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/ProjectTemplatesArgs.java:234]: We should list the possible options in the help (ds|spring).

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/petershin/liferay-portal/pull/548. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**